include/seqan3/alphabet/mask/all.hpp

```diff
@@ -23,14 +24,20 @@
  *
  * ### Introduction
  *
- * Masks are useful as tuple composites when one wants to create a masked alphabet with
- * don't care positions, but does not want to use the seqan3::dna15 **N** or
- * seqan3::aa27 **X** because of loss of information. It will instead mark the specified characters as masked,
- * and display them as lowercase representations when printed.\n
- * There are two types of masking: "hard-masking" which converts to the UNKNOWN character and
- * "soft-masking", which is visualised by using lower-case instead of upper-case.
- * However because regular nucleotide and aminoacid alphabets discard case on assignment,
- * one needs to create additional alphabets to preserve this information (if desired).\n
+ * Masks are useful as tuple composites when one wants to create a masked alphabet with don't care positions, but does
+ * not want to use the seqan3::dna15 **N** or seqan3::aa27 **X** because of loss of information. It will instead mark
+ * the specified characters as masked, and display them as lowercase representations when printed.\n
+ *
+ * There are two types of masking: "hard-masking" which converts to the UNKNOWN character and "soft-masking", which is
+ * visualised by using lower-case instead of upper-case.
+ *
+ * These usage of lower case letters is a convention popularised by ([RepeatMasker](http://www.repeatmasker.org/)) (last
+ * access 03.05.2021), where interspersed repeats (which covers transposons, retrotransposons and processed pseudogenes)
+ * and low complexity sequences are marked with lower case letters. Note that larger repeats, such as segmental
+ * duplications, large tandem repeats and whole gene duplications are generally not masked.
+ *
+ * However because regular nucleotide and aminoacid alphabets discard case on assignment, one needs to create additional
+ * alphabets to preserve this information (if desired).\n
  * This alphabet in itself is not useful to users directly, but instead the composite seqan3::masked may be used to
  * transform another alphabet into a new alphabet that can represent the original alphabet plus masking information.
```

Comment:
```suggestion
 * Masks are useful in cases where an alphabet needs to be augmented with additional information.
 * A common use case is the introduction of don't-care positions (*masking*) in \link nucleotide nucleotide \endlink or
 * \link aminoacid aminoacid \endlink sequences without using ``'N'`` or ``'X'``, respectively.
 * Instead, the original alphabet is combined with seqan3::mask to create a seqan3::masked composite.
 * When printed via the seqan3::debug_stream, masked characters are displayed in lowercase.
 *
 * \snippet TODO masked.cpp ?
 *
 * ### Types of masking
 *
 * There are two types of masking:
 *  * **hard-masking**, which converts masked characters to the ambiguous/unknown character (``'N'`` or ``'X'``)
 *  * **soft-masking**, which uses lowercase for masked characters
 *
 * ### Repeat masking
 *
 * The use of soft-masking was popularised by [RepeatMasker](https://www.repeatmasker.org/).
 * Interspersed repeats (transposons, retrotransposons and processed pseudogenes) and low complexity sequences are
 * denoted by lowercase characters. Note that larger repeats, such as segmental duplications, large tandem repeats and
 * whole gene duplications are generally not masked.
```

Can you fill in the `TODO` (link to the snippet)


I Removed access from repeatmasker and used https
(we don't really cite from there cite, so access date does not help that much)

THe part about repeat masking is very specific, so I gave it a header